README.md

```diff
@@ -258,20 +259,20 @@ spec:
-| parametersFrom | `[]object` | List of sources to populate parameters. |
-| userInfo | `object`  | Contains information about the user that last modified this service binding. |
-| credentialsRotationPolicy | `object`  | Holds automatic credentials rotation configuration. |
-| credentialsRotationPolicy.enabled | `boolean`  | Indicates whether automatic credentials rotation are enabled. |
-| credentialsRotationPolicy.rotationFrequency | `duration`  | Specifies the frequency at which the binding rotation is performed. |
-| credentialsRotationPolicy.rotatedBindingTTL | `duration`  | Specifies the time period for which to keep the rotated binding. |
+| Parameter             | Type       | Description                                                                                                                                                                                                                                                                                                                              |
+|:-----------------|:---------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| serviceInstanceName`*`   | `string`   | The Kubernetes name of the service instance to bind, should be in the namespace of the binding.                                                                                                                                                                                                                                          |
+| externalName       | `string`   | The name for the service binding in SAP BTP, defaults to the binding `metadata.name` if not specified.                                                                                                                                                                                                                                   |
+| secretName       | `string`   | The name of the secret where the credentials are stored, defaults to the binding `metadata.name` if not specified.                                                                                                                                                                                                                       |
+| secretKey | `string`  | Property of the Secret object that stores service binding data (credentials) returned from the broker. It is encoded as a JSON object to support complex data structures. [Example](#formats-of-secret-objects)                                                                                                                                     |
```

Comment:
In rootKey you use 'property' while in secretRootKey we use 'key'. key is also baked in the attributes' name. And you also use it below in the format title ("Key- Value Pairs") and the description.

"...defines a name of a key in the secret object in which the service binding data (credentials) returned from the broker will be stored..."

Reply:
@TalShorSap 
Part of the Secret object that stores service binding data (credentials) received from the broker. When secretKey is used, credentials are stored in the JSON format.

Reply:
done

Reply:
@danielforsap 
The root key... -> The secretRootKey is the key in the secret object in which... (value is json)